tedana/workflows/tedana.py

```diff
@@ -78,6 +78,18 @@ def _get_parser():
                                 "function will be used to derive a mask "
                                 "from the first echo's data."),
                           default=None)
+    optional.add_argument('--prefix',
+                          dest='prefix',
+                          type=str,
+                          help="Prefix for filenames generated.",
+                          default='')
```

Comment:
Hm, I wanted it for my own purposes as well. Do you think it's harmful to add it?
You can't make a prefix with `out_dir` because that'll create a directory with the prefix name in it.

Reply:
Maybe it would be useful outside of BIDS compatibility. Could you expand on your use-case?

Ultimately, I think this is a new argument and behavior, so we should probably float it by the group in its own issue. We talked about prefixes a _long_ time ago in #106, and decided against them, but the group was much smaller then and others may feel differently.

> Do you think it's harmful to add it?

I think there are ways to work around prefixed files in nipype/fMRIPrep, so, while it _could_ add a bit more work for BIDS Apps, I don't think it would be harmful _per se_.

Reply:
Hm, I can get rid of that for now. The short version is that I'm using AFNI, not BIDS Apps, but have files where I still have BIDS prefixes. So it would simplify some things for me as someone who's only partially in that ecosystem. Perhaps we could reopen #106 ?

Reply:
Reading #106, it looks to me like we tabled the prefix discussion to have for whenever we make the file naming more BIDS friendly (i.e. now). I was inclined to welcome something like prefix then & I still am. If everyone else things this is a relatively benign change, I'd prefer to keep it.

Reply:
If we move forward with prefix, then I'd rather merge prefix with out_dir than add a new argument. Prefix could include a path, and as long as it ends with a separator, it can fully replace out_dir.

Reply:
That's going to complicate the logic a bit and I worry that it's slightly confusing to users (this both "names" and "places" the files, rather than "this one names" and "this one places"). Is there some reason we should avoid having two arguments? We already have a ton of parameters anyway, in my view.

Reply:
`prefix` is AFNI-like, and I was under the impression that that was how the argument behaved in AFNI.

Reply:
`-prefix` is a common but very catch-all parameter that can be inconsistent in AFNI, used to variously do any of the following:
1. Place the output in a certain directory
2. Prefix a special name
3. Specify a complete name
4. Specify file type

However, I don't think we want to emulate having one option do all of those things here, particularly since we're enforcing certain naming conventions strictly. This may just be a hangup on my end so I'll ping @handwerkerd to give his thoughts, too.

Reply:
Ah, I didn't realize it was so variable.

Reply:
I weakly lean toward keeping out_dir and prefix separate. I'm fairly sure, in AFNI, when prefix is used for a directory you need to explicitly notate that. That is: `--out_dir ./newdir/identifier` is easily understandable, but `--out_dir identifier` could mean add the file prefix `identifier` or put files in an output directory called `identifier` We could require `--out_dir ./identifier/` but that seems like a confusing change in functionality.